src/App.tsx

```diff
@@ -20,12 +22,16 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={(event) => {
+                  setQuery(event.target.value);
+                }}
               />
             </div>
           </div>
         </div>
 
-        <MoviesList movies={moviesFromServer} />
+        <MoviesList movies={moviesFromServer} query={query} />
```

Comment:
You would pass `visibleMovies` instead of moviesFromServer. In that case you would not need to pass `query` here.